did-utils/src/crypto/ed25519.rs

```diff
@@ -30,10 +44,28 @@ impl KeyMaterial for Ed25519KeyPair {
 }
 
 impl Generate for Ed25519KeyPair {
+    /// Generates a new Ed25519 key pair.
+    ///
+    /// If the initial seed is empty or invalid, a random seed will be generated.
```

Comment:
What I mean here is that there is no initial seed argument.
```suggestion
    /// A random seed will be generated.
```

Reply:
This was also address in the doc ticket

Reply:
But I don't know what you did there. If caught, mistakes must be addressed where they are introduced, otherwise we risk leaving them unaddressed.

Reply:
I think I've done some mistake but for sure all the documentation related are addressed in the doc ticket.